### PR TITLE
Add Network Interface Card Details

### LINKED ISSUE #19: [Feature] Adding Network Interface Name along with the Local IPv4 Addresses
**Please describe the feature you want this project to offer**
The current output of the program for the Local IP Address is as:
GitHubIssue

**Describe the solution you'd like**

Along with the IP Address, we could Specify the Network Interface name as
 1. Ethernet:  192.168.30.1/24
 2. Wifi:  192.168.1.20/24

**Additional context**
This will make Identifying the IP of each interface simple

### PR DESCRIPTION
**Pull request by sjchoure**
Fixes #19 

- [x] Refactored `GetLocalIPAddress()` function to give details of Network Interface Cards along with IPv4, IPv6, and CIDR Block Details
- [x] Added Two Colors:

1. Bold White
2. Color Reset (Default)

## Output
GitHubPR

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -10,6 +10,8 @@ const (
 	BannerBlueColor    = "\033[1;36m%s\033[0m"
 	WarningYellowColor = "\033[1;33m%s\033[0m"
 	ErrorRedColor      = "\033[1;31m%s\033[0m"
+	BoldWhite					 = "\033[1;37m%s\033[0m"
+	None 							 = "\033[0m%s\033[0m"
```
Comment (DarthBenro008) [inline main.go]: Please run go fmt

**Review by DarthBenro008 — CHANGES_REQUESTED** [review]
Please run go fmt across the project

File: main.go (hunk 1 above)
Comment (sjchoure) [reply main.go]: Oops, sorry!

**Review by DarthBenro008 — APPROVED** [review]
LGTM

**Pull request merged**